gmplot/gmplot.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, center_lat, center_lng, zoom, map_type='', apikey=''):
         self.coloricon = os.path.join(os.path.dirname(__file__), 'markers/%s.png')
         self.color_dict = mpl_color_map
         self.html_color_codes = html_color_codes
+        self.title = 'Google Maps - gmplot'
```

Comment:
It would be convenient to add `title` as a `kwargs` parameter, so:
```python
def __init__(self, center_lat, center_lng, zoom, map_type='', apikey='', **kwargs):
    ...
    self.title = kwargs.get('title', 'Google Maps - gmplot')
```

A user would then be able to create a plotter with a title in one go:
```python
gmplot.GoogleMapPlotter(37.766956, -122.438481, 13, title='My Map')
```

Reply:
I'm not sure i agree with the use of kargs (IIRC i see other comments about it recently): the interface will just become opaque and need to dig into the code to actually know what argument we can set.

directly accessing the `title` attribute is a _lazy_ approach, if we want an argument, let's make it an optional `title='Google Maps - gmplot'` explicitly in `__initi__`

Reply:
Since `title` can't be set through the parameter list, the interface is already opaque to users who want to set a title. We should have documentation to clarify the interface. Having `title` as an explicit parameter would be a step forward, but documentation will help clarify what each parameter represents (`title` is clear enough, but `zoom` might not be (which range of values would it take on?)).

Making parameters explicit (`__init__(..., title='Google Maps - gmplot')`) would bring up the issue I mentioned in #133. It seems like using `kwargs` over explicitly-named parameters would lead to a more maintainable interface, since, for example, removing an explicit parameter would break the interface for anyone relying on the position of that parameter.

Reply:
I've merged this and split off this discussion as a separate issue: #135